### ISSUE TITLE
Forwarding command-line options to the Bela core

### ISSUE THREAD

**Issue by giuliomoro** [sensestage/supercollider#12]
in order to, e.g.: set headphones level or audio input gain

**Comment by sensestage** [comment]
arguments to add:

```
/// Whether to begin with the speakers muted
int beginMuted;
/// Level for the audio DAC output
float dacLevel;
/// Level for the audio ADC input
float adcLevel;
/// Gains for the PGA, left and right channels
float pgaGain[2];
/// Level for the headphone output
float headphoneLevel;
/// How many channels to use on the multiplexer capelet, if enabled
int numMuxChannels;
```

**Comment by sensestage** [comment]
headphone and pgaGain have been added via https://github.com/sensestage/supercollider/commit/b6fc1ef3667ab2406f84f6f3ff807246cd63b482

**Comment by giuliomoro** [comment]
great, I was wondering if it would be easier to just set something like --bela-options "<string>" and forward <string> to `Bela_getopt_long()`, similarly to what is done [here](https://github.com/BelaPlatform/Bela/blob/master/core/default_main.cpp). This would require zero effort in the future to add new options. Do you see any drawbacks?

**Comment by sensestage** [comment]
I think it is a great idea!

**Comment by sensestage** [comment]
See also #21

**Comment by sensestage** [comment]
Fixed via: 997f6253315eef97e554f9571f50cb22b8418ce0
(the other issue stays open for the new option passing)
